src/cfml/system/modules_app/server-commands/commands/server/open.cfc

```diff
@@ -48,7 +48,7 @@ component {
 			}
 			var thisURL = "#serverInfo.host#:#serverInfo.port##arguments.URI#";
 			print.greenLine( "Opening...#thisURL#" );
-			openURL( thisURL );
+			openURL( thisURL, configService.getSetting( 'preferedBrowser' ) );
```

Comment:
Instead of getting the default here, do this in the actual underlying `openURL()` method.  That way EVERY use of `openURL()` will obey the default as opposed to just this one.